### ISSUE TITLE
Add possibility to add content via other apps

### ISSUE THREAD

**Issue by fourlastor** [danybony/magic-mirror-dashboard#3]
Something similar to (Muzei)[https://github.com/romannurik/muzei/], so that one wouldn't have to fork the app to add functionality.

What do you think? 💃

**Comment by danybony** [comment]
Good idea, I'll investigate a bit more how Muzei worked to evaluate the required effort.
